python-modules/cis_identity_vault/setup.py

```diff
@@ -6,7 +6,7 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-requirements = ["everett", "everett[ini]", "configobj", "boto", "boto3", "botocore", "sqlalchemy", "psycopg2"]
+requirements = ["everett", "everett[ini]", "configobj", "boto", "boto3", "botocore", "sqlalchemy", "sqlalchemy_utils"]
```

Comment:
as per error msg during import, it sounds like this should include both sqlalchemy_utils and psycopg2